TextReader.java

```diff
@@ -1 +1,79 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
 
+public class TextReader {
+
+	private ArrayList<String> lines = new ArrayList<String>();
+	Map<String, ArrayList<Integer>> wordMap = new HashMap<>();
+
+	public void read(String fileName, String pattern) {
+
+		setUpWordMap(fileName);
+
+		Set<Integer> matchedLines = matchWords(wordMap, pattern);
+
+		printResults(matchedLines,lines);
```

Comment:
Added space between "," and "lines"
```suggestion
		printResults(matchedLines, lines);
```